_docs-sources/developer-portal/faq.md

```diff
@@ -0,0 +1,65 @@
+# FAQ
+
+## How can a user change their GitHub account(unlink/link)?
+
+1. Go to [https://app.gruntwork.io/settings/profile](https://app.gruntwork.io/settings/profile)
+1. Click the **Unlink** link in the &ldquo;GitHub Account&rdquo; section as shown below
+    ![Unlink GitHubId on Profile page](/img/faq/developer-portal/unlink-github.png)
+1. [Follow these steps](./link-github-id.md) to link a new GitHub account *using a private/incognito browser window*.
+    :::tip
+
+    A private/incognito browser window guarantees you’ll have the opportunity to specify the new Github account you wish to link.
+
+    :::
+
+## How can a user change the email associated with an account?
+
+**Emails cannot be changed.**
+
+Emails serve as the primary authentication & identification mechanisms — essentially functioning as a username. For both of these reasons, changing them is currently unsupported.
+
+If a you need to use a different email (for instance, if you were using a personal email before, or your company domain changed), then:
+
+1. You or an admin on your team will need to send invitations to the new email(s).
+1. Accept the invitation for the new email to create a new user and sign in.
+1. Link your GitHub account to the new user profile when prompted after sign in.
+    :::tip
+
+    If you link the same GitHub account you had on the older user account to the new user account prior to the old account being revoked from the team, your access to our GitHub Organization will be unaffected.
```

Comment:
I eventually got there, but this sentence is a bit of brain teaser :) maybe it would be simpler if split up into two, one for each scenario or outcome (lost access / won't lose access)?

Reply:
@zackproser I revised this without breaking it up. Is it good enough?